config/default.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+return [
```

Comment:
If we do configuration in PHP I want it to be in an object orientated way:

```
return (new ProjectConfiguration())
  ->setTests('tests/phel/')
  ->setExport(
    (new ExportConfiguration())
      ->setDirectories(['src/phel'])
      ->setNamespacePrefix('PhelGenerated')
      ->setTargetDirectory('src/PhelGenerated')
   );
```

Is that possible?